packages/core/src/sheets/view-model.ts

```diff
@@ -71,7 +72,23 @@ export class SheetViewModel extends Disposable {
 
     getCell(row: number, col: number): Nullable<ICellDataForSheetInterceptor> {
         if (this._cellContentInterceptor) {
-            return this._cellContentInterceptor.getCell(row, col);
+            return this._cellContentInterceptor.getCell(row, col, InterceptorEffectEnum.Value | InterceptorEffectEnum.Style);
+        }
+
+        return this.getRawCell(row, col);
+    }
+
+    getCellValueOnly(row: number, col: number): Nullable<ICellDataForSheetInterceptor> {
+        if (this._cellContentInterceptor) {
+            return this._cellContentInterceptor.getCell(row, col, InterceptorEffectEnum.Value);
+        }
+
+        return this.getRawCell(row, col);
+    }
+
+    getCellStyleOnly(row: number, col: number): Nullable<ICellDataForSheetInterceptor> {
```

Comment:
@lumixraku  here , after merged this pr, you can use above two api in sketon to get better performance

Reply:
merge now?